infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -885,7 +908,9 @@ private LogEntry getLogEntry(long address, LogData entry) {
             }
 
             allRecordsBuf.flip();
-            safeWrite(segment.getWriteChannel(), allRecordsBuf);
+            // This API is called by stateTransfer, is that considered a high priority
```

Comment:
is that -> that is

Reply:
Is it a high priority or normal priority? The comment and code are inconsistent.

Reply:
I moved the priority to the msg level. We should take all requests with a client's priority level.